### ISSUE TITLE
G code start and end configure

### ISSUE THREAD

**Issue by cfoxwood60** [Ultimaker/Cura#11579]
### Application Version

4.13.1

### Platform

Windows 10

### Printer

Creality ender 3

### Reproduction steps

Tried to copy and paste start and end programs into machine settings but it will not let me . I have to enter the code manually.

### Actual results

The copy and paste file does not appear

### Expected results

The pasted file should appear in the box

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Ender 3 V2 Start G-code
; Ender 3 Custom Start G-code
G92 E0 ; Reset Extruder
G28 ; Home all axes
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little
G1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X5 Y20 Z0.3 F5000.0 ; Move over to prevent blob squish

**Comment by GregValiant** [comment]
Yes, there is no right-click menu in the Startup or End gcode boxes.  You should be able to copy with CTRL+C and paste with CTRL+V.

**Comment by cfoxwood60** [comment]
Thanks for the reply. I ended up doing it that way. Once again thankyou.

**Comment by GregValiant** [comment]
On a personal note -I have found the lack of a right click menu to be annoying.  The CTRL buttons on this old laptop are glitchy and using them to copy and paste often makes a mess.

**Comment by fvrmr** [comment]
Hi @cfoxwood60 thank you for your report. 
This is then a new feature, I will bring it up with the team.
Keep you posted.

**Comment by fvrmr** [comment]
I have discussed this with the team. And I created a ticket on our backlog. 
Devs see CURA-9051
